deps/patches/gmp_alloc_overflow_func.patch

```diff
@@ -0,0 +1,230 @@
+diff --git a/gmp-h.in b/gmp-h.in
+--- a/gmp-h.in
++++ b/gmp-h.in
+@@ -477,10 +477,17 @@ using std::FILE;
+ #define mp_get_memory_functions __gmp_get_memory_functions
+ __GMP_DECLSPEC void mp_get_memory_functions (void *(**) (size_t),
+ 				      void *(**) (void *, size_t, size_t),
+ 				      void (**) (void *, size_t)) __GMP_NOTHROW;
+ 
++#define mp_set_alloc_overflow_function __gmp_set_alloc_overflow_function
++__GMP_DECLSPEC void mp_set_alloc_overflow_function (void (*) (void)) __GMP_NOTHROW;
++
++#define mp_get_alloc_overflow_function __gmp_get_alloc_overflow_function
++__GMP_DECLSPEC void mp_get_alloc_overflow_function (void (**) (void)) __GMP_NOTHROW;
```

Comment:
Is there a reason not to use a return value here?

Reply:
It's just to keep similar style with mp_{get,set}_memory_functions